src/Main/ReplaceGameElements/NativeGrimoire.js

```diff
@@ -13,7 +13,7 @@ import ReplaceTooltipGrimoire from './TooltipGrimoire';
  * This function fixes replaces the .draw function of the Grimoire
  */
 function ReplaceNativeGrimoireDraw() {
-  if (!HasReplaceNativeGrimoireDraw && Game.Objects['Wizard tower'].minigameLoaded) {
+  if (Game.drawT % 5 === 0 && !HasReplaceNativeGrimoireDraw && Game.Objects['Wizard tower'].minigameLoaded) {
```

Comment:
Why is this needed?

Reply:
The game now draws the mana bar text every 5 frames instead of every frame. without this change, CM will append the timer 5 times before the game resets it

Reply:
This has been fixed in another PR! See #976